### PR TITLE
update hypershift cli repos

### PR DESCRIPTION
**Pull request by mukrishn**
### Description
To support any custom fork thru vars for these CLIs and also pick a particular release/tag.
```
    "rosa_cli_fork": "https://github.com/openshift/rosa",
    "ocm_cli_fork": "https://github.com/openshift-online/ocm-cli",
    "hypershift_cli_fork": "https://github.com/openshift/hypershift"
    "ocm_cli_version": "master",
    "hypershift_cli_version": "main",
    "rosa_cli_version": "master",
```

### CODE REVIEW

**Comment by mukrishn** [comment]
@dry923 PTAL, this change is based on your [suggestion](https://github.com/cloud-bulldozer/airflow-kubernetes/pull/167#discussion_r854523830) to include configurable CLI repos.

File: dags/openshift_nightlies/scripts/install/hypershift.sh [hunk 1]
```diff
@@ -53,8 +53,9 @@ setup(){
     export KUBECONFIG=/home/airflow/auth/config
     if [[ $INSTALL_METHOD == "osd" ]]; then
         export OCM_CLI_VERSION=$(cat ${json_file} | jq -r .ocm_cli_version)
-        if [[ ${OCM_CLI_VERSION} == "master" ]]; then
-            git clone https://github.com/openshift-online/ocm-cli
+        if [[ ${OCM_CLI_VERSION} != "null" ]]; then
+            OCM_CLI_FORK=$(cat ${json_file} | jq -r .ocm_cli_fork)
```
Comment (dry923) [inline dags/openshift_nightlies/scripts/install/hypershift.sh:57]: @mukrishn Where is ocm_cli_fork (and rosa_cli_fork, etc) defined? I'm not seeing it as a var in the .json files

Comment (dry923) [reply dags/openshift_nightlies/scripts/install/hypershift.sh:57]: oooooh they are in the defaults. disregard

**Review by dry923 — APPROVED** [review]
LGTM!

**Pull request merged**